examples/CMakeLists.txt

```diff
@@ -0,0 +1,63 @@
+# Minimum CMake required

```

Comment:
Both. examples/CMakeLists.txt is a stand alone that is meant to be installed with the examples, while cmake/examples.cmake is a series of calls to ExternalProject_Add that reference it and let you build the examples as part of your build of the protobuf project for ease of testing.